src/tests/ftest/server/metadata.py

```diff
@@ -62,10 +61,14 @@ class ObjectMetadata(TestWithServers):
     """
 
     # Minimum number of containers that should be able to be created
-    CREATED_CONTAINERS_MIN = 2900
+    CREATED_CONTAINERS_MIN = 25000
```

Comment:
Could we add container to be created to metadata.yaml?

Reply:
I can look into this, may need some assistance if I can't figure it out based on looking at other similar examples in the code.

Reply:
It works similar to this config:
https://github.com/daos-stack/daos/blob/042ffb9c9776757e59ec5424ad95d13eb34e600f/src/tests/ftest/server/metadata.yaml#L65-L66
And to get it in the test code:
https://github.com/daos-stack/daos/blob/042ffb9c9776757e59ec5424ad95d13eb34e600f/src/tests/ftest/server/metadata.py#L355
And that same location in the config is where it should go

Reply:
added both parameters into that section of metadata.yaml in the latest code on the branch.